File: javascript-closure-angular-client/API/Client/LineForGoodsCalculatedTax.js
goog.provide('API.Client.LineForGoods_calculatedTax');

/**
 * @record
 */
API.Client.LineForGoodsCalculatedTax = function() {}

/**
 * @type {!API.Client.LineForGoods_calculatedTax_taxByType}
 * @export
 */
API.Client.LineForGoodsCalculatedTax.prototype.taxByType;

/**
 * @type {!number}
 * @export
 */
API.Client.LineForGoodsCalculatedTax.prototype.tax;

/**
 * @type {!Array<!API.Client.DetailsCalculatedTaxItem>}
 * @export
 */
API.Client.LineForGoodsCalculatedTax.prototype.details;

File: async-scala-client/src/main/scala/io/swagger/client/model/LineForGoods_calculatedTax.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class LineForGoods_calculatedTax (
  taxByType: Option[LineForGoods_calculatedTax_taxByType],
tax: Option[Double],
details: Option[List[DetailsCalculatedTaxItem]]
)
